scripts/lib/setup/installers.sh

```diff
@@ -319,6 +345,30 @@ function use_android_sdk() {
 }
 
 function install_android_ndk() {
+  if is_macos; then
+    install_android_ndk_mac
+  elif is_linux; then
+    install_android_ndk_linux
+  fi
+}
+
+function install_android_ndk_linux() {
+  if [ -e ~/Android/android-ndk-r10e/ ]; then
+    cecho "@green[[+ Skipping Android NDK setup- the ~/Android/android-ndk-r10e/ already exists ]]"
+  else
+    cd ~/Android/
+    wget http://dl.google.com/android/repository/android-ndk-r10e-linux-x86_64.zip
+    unzip android-ndk-r10e-linux-x86_64.zip
+    cd android-ndk-r10e/
+    export ANDROID_NDK=$(pwd)
```

Comment:
We could keep it consistent with the previous `install_android_ndk` implementation, where instead of changing folders and using `$(pwd)`, we define a `local _ndkParentDir=~/Android/android-ndk-r10e/` instead.